management/publishSettings.go

```diff
@@ -7,7 +7,7 @@ import (
 	"fmt"
 	"io/ioutil"
 
-	"github.com/Azure/go-pkcs12"
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12"
```

Comment:
also keep in mind that if some project imports this package + `golang.org/x/crypto/pkcs12` separately, then it'll end up two copies of it, possibly even duplicate types with the same name in the runtime.